### PR TITLE
[DM-52] Add generated image in templates

### PR DESCRIPTION
**Pull request by VicThorX**
Hi team!

Here I will add the image generated by Relay.
I modify some CSS styles to adapt the image on the template rectangle. (If someone can help me with CSS to make the image fit well in the template rectangle, my CSS is very rusty)

If anyone has suggestions, please comment!

Relay PR (https://github.com/MakingSense/Doppler_Relay/pull/1643)

### CODE REVIEW

**Comment by andresmoschini** [comment]
Nice! Just one thing, if the property `template.thumbnail` is empty, are you ready to show a default template?

**Comment by VicThorX** [comment]
> Nice! Just one thing, if the property `template.thumbnail` is empty, are you ready to show a default template?

Yes, I will put a default image.

File: src/package.json [hunk 1]
```diff
@@ -43,8 +43,8 @@
     "karma-chrome-launcher": "^2.0.0",
     "karma-jasmine": "~0.3.6",
     "karma-phantomjs-launcher": "^1.0.2",
-    "protractor": "5.1.0",
-    "selenium-webdriver": "3.0.1"
+    "protractor": "^5.1.0",
+    "selenium-webdriver": "^3.0.1"
```
Comment (elsupergomez) [inline src/package.json]: Just curious, why change this?

Comment (VicThorX) [reply src/package.json]: That should not be here!
I changed it in my local pc to can generate the build and test.

Comment (VicThorX) [reply src/package.json]: But, I still not know why with Jenkins the tests fail

**Comment by andresmoschini** [comment]
👏

**Pull request merged**